static/app/utils/profiling/frame.tsx

```diff
@@ -13,7 +13,16 @@ export class Frame extends WeightedNode {
   resource?: string;
   recursive?: Frame;
 
-  constructor(frameInfo: Profiling.FrameInfo, type: 'mobile' | 'web') {
+  static Root = new Frame(
+    {
+      key: 'sentry root',
+      name: 'sentry root',
```

Comment:
We do we name the root frame like this?

also this can be done in another PR, but could you add comments describing each of the fields in `frame`? It helps with understanding the logic that happens below.

Reply:
@AbhiPrasad this is just to have a root frame, note that this doesn't ever get displayed anywhere and just serves as the entry point for the tree.